comms/storage/storageserver/weather-map/src/routes/History.tsx

```diff
@@ -0,0 +1,152 @@
+import dayjs from 'dayjs'
+import { useEffect, useState } from 'react'
+import { useSearchParams } from 'react-router-dom'
+
+import { useStatusUpdateLogs } from '../api'
+import DateTimePicker from '../components/dateTimePicker/DateTimePicker'
+
+const FORMAT = 'YYYY-MM-DD hh:mm:ss A'
+type TimelineItem = {
+  datetime: dayjs.Dayjs
+  content: string // TODO: Add content-collapsible to show details like diff of shards stored or num files rebalanced
+  icon: JSX.Element
+  iconBg: string
+}
+
+// TODO: Add events: updateHealthyNodeSet, rebalanceStart, rebalanceEnd
+// TODO: Add select dropdowns to filter by node
+// TODO: Make time hoverable to show UTC
```

Comment:
this is the core react component to visualize. these TODOs are for the next PR since this one is big enough